### ISSUE TITLE
Issue getting this setup!

### ISSUE THREAD

**Issue by vsoch** [yandex-cloud/k8s-csi-s3#40]
Hi! I've been trying a few days to get this working, and aside from probably being an idiot I suspect I'm doing something wrong. My use case is that I have an existing bucket that I want to attach pods to (in an indexed job) to use the workflow flies. What I've tried:

- I started with the static example, but it doesn't work to provide an empty storageclass because the error message is that it doesn't know the class to use.
- I then thought I was getting close, but it could not find a mounter named geesefs. I can't find instructions anywhere how to install / set that up, I assumed it came with the CSI here.
- I tried changing to the other mounter type, that failed even earlier because the PVC couldn't create with "DeadlineExceeded desc = context deadline exceeded"

I just want to mount my bucket on S3. Please help.

**Comment by vitalif** [comment]
Hi. The second point suggests that maybe you just have another version of CSI S3 installed? Did you install it from this exact repository? It's a fork of other version so other container registries may have other version.

**Comment by vsoch** [comment]
I was using `ru.yandex.s3.csi ` as the driver - is there another one?

**Comment by vitalif** [comment]
No, it was the correct one. I don't know how it couldn't find geesefs mounter in your case. You were doing something wrong, but I'm not sure what :)

**Comment by vsoch** [comment]
haha well you know, that sort of is my forte :P 

I'm planning on trying this again as soon as I get more aws credits!
